### PR TITLE
feat: add  python examples

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <zhuhan@cestc.cn>

ref: https://github.com/CECTC/dbpack/issues/31

### Ⅰ. Describe what this PR did
add  python examples

### Ⅱ. Does this pull request fix one issue?
https://github.com/CECTC/dbpack/issues/31

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
No need

### Ⅳ. Describe how to verify it
See README

### Ⅴ. Special notes for reviews
None.

### CODE REVIEW

**Comment by zhu733756** [comment]
I have read the CLA Document and I hereby sign the CLA

File: samples/python/config2.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+filters:
+  - name: MysqlDistributedTransaction
+    conf:
+      appid: productSvc
+      lock_retry_interval: 100ms
+      lock_retry_times: 15
+
+data_source_cluster:
+  - name: seata_product
+    capacity: 30
```
Comment (dk-lockdown) [inline samples/python/config2.yaml:27]: pls change datasource name to `product`

File: samples/python/config2.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      users:
+        dksl: "123456"
+      server_version: "8.0.27"
+    executor: redirect
+
+executors:
+  - name: redirect
+    mode: sdb
+    config:
+      data_source_ref: seata_product
```
Comment (dk-lockdown) [inline samples/python/config2.yaml]: pls change datasource name to `product`

File: samples/python/config3.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      users:
+        dksl: "123456"
+      server_version: "8.0.27"
+    executor: redirect
+
+executors:
+  - name: redirect
+    mode: sdb
+    config:
+      data_source_ref: seata_order
```
Comment (dk-lockdown) [inline samples/python/config3.yaml]: pls change datasource name to `order`

File: samples/python/config3.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+filters:
+  - name: MysqlDistributedTransaction
+    conf:
+      appid: orderSvc
+      lock_retry_interval: 100ms
+      lock_retry_times: 15
+
+data_source_cluster:
+  - name: seata_order
```
Comment (dk-lockdown) [inline samples/python/config3.yaml]: pls change datasource name to `order`

**Comment by zhu733756** [comment]
OK.  This feature still needs further impoved and under WIP

**Comment by zhu733756** [comment]
Done @dk-lockdown  Pls take a look.

**Review by dk-lockdown — APPROVED** [review]
lgtm

**Comment by zackzhangkai** [comment]
lgtm

**Review by bohehe — APPROVED** [review]
lgtm

**Pull request merged**